components/permissions/contexts/brave_ethereum_permission_context.cc

```diff
@@ -47,7 +47,7 @@ BraveEthereumPermissionContext::BraveEthereumPermissionContext(
 BraveEthereumPermissionContext::~BraveEthereumPermissionContext() = default;
 
 bool BraveEthereumPermissionContext::IsRestrictedToSecureOrigins() const {
-  return true;
+  return false;
```

Comment:
Can you add the descriptions you have in the PR of why we need this as a comment here please? Thanks.

Reply:
fixed